.storybook/main.js

```diff
@@ -1,4 +1,5 @@
 const path = require('path');
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
```

Comment:
using the plugin to avoid copying alias values in three different places :) need to have them in `tsconfig.json` and webpack for host and storybook.